src/tests/ftest/container/attribute.py

```diff
@@ -68,44 +86,62 @@ def verify_list_attr(self, indata, size, buff, mode="sync"):
                 "len={} and received len={}".format(length, size))
         # verify the Attributes names in list_attr retrieve
         for key in list(indata.keys()):
-            if key not in buff:
+            found = False
+            for attr in attributes_list:
+                # Workaround
+                # decode() is used to be able to compare bytes with strings
+                # We get strings from daos_command, while in pydaos we get bytes
+                if key.decode() == attr:
+                    found = True
+                    break
+            if not found:
                 self.fail(
                     "FAIL: Name does not match after list attr, Expected "
-                    "buf={} and received buf={}".format(key, buff))
+                    "buf={} and received buf={}".format(key, attributes_list))
 
     def verify_get_attr(self, indata, outdata):
         """
         verify the Attributes value after get_attr
         """
         self.log.info("Verifying get_attr output:")
         self.log.info("  get_attr data: %s", indata)
-        self.log.info("  set_attr date: %s", outdata)
+        self.log.info("  set_attr data: %s", outdata)
 
         for attr, value in indata.items():
+            # Workaround: attributes from daos_command have b prefix
+            # need to remove it
             if value != outdata[attr]:
                 self.fail(
                     "FAIL: Value does not match after get attr, Expected "
-                    "val={} and received val={}".format(value, outdata[attr]))
+                    "val={} and received val={}".format(value,
+                                                        outdata[attr]))
 
     def test_container_large_attributes(self):
         """
         Test ID: DAOS-1359
 
         Test description: Test large randomly created container attribute.
 
-        :avocado: tags=container,container_attr,attribute,large_conattribute
+        :avocado: tags=container,attribute,large_conattribute
+        :avocado: tags=container_attribute
         """
         self.add_pool()
         self.add_container(self.pool)
         self.container.open()
-
+        self.daos_cmd = DaosCommand(self.bin)
         attr_dict = self.create_data_set()
 
         try:
             self.container.container.set_attr(data=attr_dict)
-            # skip step due to DAOS-6880
-            #size, buf = self.container.container.list_attr()
-            #self.verify_list_attr(attr_dict, size, buf)
+
+            # Workaround
+            # Due to DAOS-7093 skip the usage of pydaos cont list attr
+            # size, buf = self.container.container.list_attr()
```

Comment:
is there a ticket to re-enable this?

Reply:
Yes: https://jira.hpdd.intel.com/browse/DAOS-7093

Reply:
Well, DAOS-7093 doesn't say that we would need to uncomment line L139 :)

Reply:
Ticked was updated adding the information about container.list_attr() line.